packages/notifications/push/amplify_push_notifications/android/src/main/kotlin/com/amazonaws/amplify/amplify_push_notifications/AmplifyPushNotificationsPlugin.kt

```diff
@@ -168,6 +175,14 @@ open class AmplifyPushNotificationsPlugin : FlutterPlugin, ActivityAware,
         return true
     }
 
+    override fun registerCallbackFunction(
+        callbackHandle: Long,
```

Comment:
Is `Int` not enough to contain the callback handle?

Reply:
It is.